### DISCUSSION TITLE
Install eclipses mqtt.paho.cpp as static library

### DISCUSSION THREAD

**Discussion by avvocato-sudo** [eclipse-paho/paho.mqtt.c#1478] [Q&A]
Hi I try for a while now to get the mqtt.paho.cpp from eclispe on my machine (windows 11) to use it as a static library for project in QNX (embeded).
I follow the step as told in the readme of the repository, but when i get to the point to call the follwoing command on cmd:
cmake -Bbuild -H. -DCMAKE_INSTALL_PREFIX=C:\mqtt\paho-cpp -DPAHO_BUILD_SAMPLES=ON -DPAHO_WITH_SSL=OFF -DCMAKE_PREFIX_PATH=C:\mqtt\paho-c
I always get the following error
-- Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.22631.
-- Configuring done (0.0s)
CMake Error at src/CMakeLists.txt:84 (target_link_libraries):
  Target "paho-mqttpp3-static" links to:

    eclipse-paho-mqtt-c::paho-mqtt3a-static

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

-- Generating done (0.1s)
CMake Generate step failed.  Build files cannot be regenerated correctly.

I have absolutely no clue about cmake, so i would appreciate some help.
Thx!

**Comment by fpagliughi** [comment]
Try this with the latest releases - C++ v1.5.0 with C v1.3.14. These were updated this week with a number of fixes to the CMake build to fix these issues. If there's still a problem with these versions, open an issue on the C++ repo.
